src/explore-education-statistics-common/src/components/form/FormComboBox.tsx

```diff
@@ -52,7 +52,6 @@ const FormComboBox = ({
   const listBoxRef = useRef<HTMLUListElement>(null);
   const inputRef = useRef<HTMLInputElement>(null);
   const optionRefs = useRef<Dictionary<HTMLLIElement>>({});
-
```

Comment:
Nitpick, but no need to remove the spacing here. The refs were logically spaced out from the state